publishable/config/voyager.php

```diff
@@ -94,21 +95,19 @@
     */
 
     'widgets' => [
-        [
-            'name'  => 'User',
+        'user' => [
             'icon'  => 'voyager-group',
             'model' => TCG\Voyager\Models\User::class,
             'url'   => 'admin/users',
             'image' => '/images/widget-backgrounds/02.png',
         ],
```

Comment:
i think the current one looks good & easy to understand
```php
'user' => [ // the name of the widget
     'icon'  => 'voyager-group', // the icon you want this widget to have  
     'model' => TCG\Voyager\Models\User::class, // the class responsible for the widget logic
     'url'   => 'admin/users', // where this widget should re-direct to when clicked
     'image' => '/images/widget-backgrounds/02.png', // background for the widget box
 ],
```

the good thing about this is, if the user don't want any of them he can simply remove/comment it, or if he wants to add a new one he just duplicate and start adding his config,
even the model is not enforced as he just needs to add the full namespace of the class and it will be picked up.

i understand that u want to keep this as minimal as possible but over simplifying things will lead to the exact opposite.